src/lib/assetMiddleware.js

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk"
+import fs from "fs"
+import { isProduction } from "./environment"
+import path from "path"
+
+const { CDN_URL } = process.env
+
+export const assetMiddleware = () => {
```

Comment:
Before adding the asset middleware we'll want to make sure the current webpack stuff in master is working 100%. In force I did things in two phases; 1) make webpack setup a little friendlier to changes over time; 2) remove bucket assets. 

These were the stages (with maybe a few intermediate prs inbetween that fixed issues): 
1) https://github.com/artsy/force/pull/3434
2) https://github.com/artsy/force/pull/3458
3) https://github.com/artsy/force/pull/3630

I would reference the Arena PR linked in the ticket, however. It's straight forward and to the point.